src/widget-loader.ts

```diff
@@ -286,14 +286,7 @@ export function loadTemplates(settings: EnforcedWidgetSettings) {
         return
       }
 
-      const { styles, template } = customTemplate
-
-      if (styles) {
-        styles.forEach(style => {
-          const { css } = style
-          sdk.placement.injectStaticComponentStyle(key, css)
-        })
-      }
+      const { template } = customTemplate
```

Comment:
Should you also remove it from here 
https://github.com/Stackla/widget-utils/blob/master/src/widget-loader.ts#L99